test/e2e/start_test.go

```diff
@@ -249,4 +249,71 @@ var _ = Describe("Podman start", func() {
 		Expect(session1).Should(Exit(0))
 		Expect(session1.OutputToString()).To(BeEquivalentTo(cid2))
 	})
+
+	It("podman start container sets HOME to home of caller if HOME is mounted", func() {
+		home, _ := os.UserHomeDir()
+		session := podmanTest.Podman([]string{"create", "--user", "root", "--volume", fmt.Sprintf("%s:%s:ro", home, home), ALPINE, "ls"})
+		session.WaitWithDefaultTimeout()
+		cid := session.OutputToString()
+		session = podmanTest.Podman([]string{"start", cid})
+		session.WaitWithDefaultTimeout()
+		session = podmanTest.Podman([]string{"inspect", cid, "--format", "{{.Config.Env}}"})
+		session.WaitWithDefaultTimeout()
+		env := session.OutputToString()
+		Expect(env).To(ContainSubstring(fmt.Sprintf("HOME=%s", home)))
+
+		session = podmanTest.Podman([]string{"stop", cid})
+		session.WaitWithDefaultTimeout()
+
+		session = podmanTest.Podman([]string{"start", cid})
+		session.WaitWithDefaultTimeout()
+		session = podmanTest.Podman([]string{"inspect", cid, "--format", "{{.Config.Env}}"})
+		session.WaitWithDefaultTimeout()
+		env = session.OutputToString()
+		Expect(env).To(ContainSubstring(fmt.Sprintf("HOME=%s", home)))
+	})
+
+	It("podman start container sets HOME to home of execUser if HOME of caller is not mounted", func() {
```

Comment:
test already passed on main, is this expected?

Reply:
Only the last test was meant to pass on main. 2/3 tests should now fail on main.